source/grid/grid_tools.cc

```diff
@@ -2662,7 +2662,7 @@ namespace GridTools
     static inline std::bitset<3> lookup (const MATCH_T &)
     {
       // The 1D case is trivial
-      return 4; // [true ,false,false]
+      return 1; // [true ,false,false]
```

Comment:
As I said, I'm not sure that my patch is right - I merely tried to reproduce what 2D/3D give for the standard case